code/modules/power/tesla/energy_ball.dm

```diff
@@ -205,7 +205,7 @@
 	if(!(zap_flags & ZAP_ALLOW_DUPLICATES))
 		LAZYSET(shocked_targets, source, TRUE) //I don't want no null refs in my list yeah?
 	. = source.dir
-	if(power < 1000)
+	if(power < 4e5)
```

Comment:
@Pickle-Coding This has broken tesla coils. They reach this part with power of 20000, which is always false.

Reply:
The power depends on how much is in the powernet, however I think I accidentally raised the requirement.

Tesla coils use to store energy in their weird power unit, so that got scaled. However the tesla zapping was based on the grid without the power scaling, so the cutoff point raised while tesla zap power didn't (because it shouldn't).

I think the fix is to create a zap cutoff var for tesla zaps and make tesla coil zaps use the old value.

Reply:
The damage scaling also needs to get fixed, but the solution is uglier. I got a fix up for the cutoff point so far.

Reply:
Please consider flicking the tesla animation when you pulse the wire. If there is not enough power in the network, at least it will show the animation.

Reply:
I might not be able to fix the tesla coil zap damage scaling soon, so free gbp (though kinda tedious task) oppportunity. Haven't setup my development environment yet, and I need to do some things.